images/openemr/Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM registry.access.redhat.com/ubi8 as builder
 
 RUN dnf update -y
-RUN dnf install -y @php php-mysqlnd php-soap php-gd php-pecl-zip wget git npm
+RUN dnf install -y @php php-mysqlnd php-soap php-gd php-pecl-zip wget git npm && dnf clean all
```

Comment:
`dnf clean all` is not strictly necessary as we have multistaged build. Although it does not hurt.

It is basically question, whether we want to use builder's CPU&io to do the cleaning *or* whether we want to use builder's space to keep stage layers.

Reply:
Agreed for the builder, but wouldn't it still help in the image build stage?